components/benefit_cards_b.js

```diff
@@ -332,7 +337,7 @@ export class BenefitCardB extends Component {
               expanded={this.state.open}
               className={
                 this.state.open
-                  ? classes.ExpansionPanelOpen
```

Comment:
So can we 🔥 the `ExpansionPanelOpen` class in the style section?

Reply:
Good call!